### DISCUSSION TITLE
File download

### DISCUSSION THREAD

**Discussion by baahar** [etsy/open-api#754] [Ideas]
Hi,

The API provides URLs for full images and video files of listings and so allows third-party apps to download them. Unfortunately it does not provide URLs for associated digital files. 

Is this a planned feature perhaps?

**Comment by nateswart** [comment]
This topic is being discussed, but there are no firm announcements to make at this time. We will include your comments for consideration. Thanks!

**Reply by JoannSellerServices** [reply]
This would be great!  Our application has an import listings feature that get's held up by digital downloads because we can't actually retrieve the file.  We are able to push new listing digital downloads, but importing existing ones is a difficult problem that we don't currently have a workaround for.

**Comment by ambikesh-cedcommerce** [comment]
Hi @nateswart, 
Is this fixed? I'm also facing the same issue while importing the file ( Can't download it). 

Any help would be appreciated
Thanks

**Comment by Adushar** [comment]
Hey, I am having the same issue. @nateswart if you could take a look it would be great

**Comment by PapaCharlie81** [comment]
I am suspecting that this API endpoint exists but it not made public as this would be a huge feature for people who started on Etsy and are trying to expand to other marketplaces so maybe it is not one that Etsy wants to be utilized. 
Does anyone know if the "Pattern by Etsy" is able to sync digital listing files?
